### ISSUE TITLE
python3 -m markdown doesn't work

### ISSUE THREAD

**Issue by mitya57** [Python-Markdown/markdown#91]
Comment in `markdown/__main__.py` mentions ability to use `-m markdown` option with Python 3.x:

```
# Python 2.7 & 3.x do: `python -m markdown [options] [args]`.
```

But it doesn't work:

```
Traceback (most recent call last):
  File "/usr/lib/python3.2/runpy.py", line 160, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python3.2/runpy.py", line 73, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3/dist-packages/markdown/__main__.py", line 87, in <module>
    run()
  File "/usr/lib/python3/dist-packages/markdown/__main__.py", line 81, in run
    markdown.markdownFromFile(**options)
  File "/usr/lib/python3/dist-packages/markdown/__init__.py", line 426, in markdownFromFile
    kwargs.get('encoding', None))
  File "/usr/lib/python3/dist-packages/markdown/__init__.py", line 364, in convertFile
    output_file.write(html)
  File "/usr/lib/python3.2/codecs.py", line 356, in write
    self.stream.write(data)
TypeError: must be str, not bytes
```

Package version: 2.1.1.

**Comment by waylan** [comment]
This should have been fixed with issue #79. Try pulling the latest code from git and it should work.

**Comment by mitya57** [comment]
Thanks, I should have done that before opening the ticket :)
